testfiles/PlantLoopHeatPump_EIR_AirSource_and_AWHP.idf

```diff
@@ -0,0 +1,711 @@
+    0.0,                     !- Coefficient2 x
+    0.0,                     !- Coefficient3 x**2
+    -17.77778,               !- Minimum Value of x !- 0 F
+    35.0,                    !- Maximum Value of x !- 95 F
+    -100.0,                  !- Minimum Curve Output
+    35.0,                    !- Maximum Curve Output
+    Temperature,             !- Input Unit Type for X
+    Temperature;             !- Output Unit Type
+
+  Curve:Quadratic,
+    MaxCoolingSWTvsOAT,      !- Name
+    53.1666666666667,        !- Coefficient1 Constant
+    0.85,                    !- Coefficient2 x
+    0.0,                     !- Coefficient3 x**2
+    -17.777778,              !- Minimum Value of x !- 0 F
+    35.0,                    !- Maximum Value of x !- 95 F
+    20.0,                    !- Minimum Curve Output
+    60.0,                    !- Maximum Curve Output
+    Temperature,             !- Input Unit Type for X
+    Temperature;             !- Output Unit Type
+
+  Curve:Biquadratic,
+    CapCurveFuncTemp,        !- Name
+    0.5,                     !- Coefficient1 Constant
+    0.0,                     !- Coefficient2 x
+    0.0,                     !- Coefficient3 x**2
+    0.0,                     !- Coefficient4 y
+    0.0,                     !- Coefficient5 y**2
+    0.0,                     !- Coefficient6 x*y
+    5.0,                     !- Minimum Value of x
+    10.0,                    !- Maximum Value of x
+    24.0,                    !- Minimum Value of y
+    35.0,                    !- Maximum Value of y
+    ,                        !- Minimum Curve Output
+    ,                        !- Maximum Curve Output
+    Temperature,             !- Input Unit Type for X
+    Temperature,             !- Input Unit Type for Y
+    Dimensionless;           !- Output Unit Type
+
+  Curve:Biquadratic,
+    EIRCurveFuncTemp,        !- Name
+    1.0,                     !- Coefficient1 Constant
+    0.0,                     !- Coefficient2 x
+    0.0,                     !- Coefficient3 x**2
+    0.0,                     !- Coefficient4 y
+    0.0,                     !- Coefficient5 y**2
+    0.0,                     !- Coefficient6 x*y
+    5.0,                     !- Minimum Value of x
+    10.0,                    !- Maximum Value of x
+    24.0,                    !- Minimum Value of y
+    35.0,                    !- Maximum Value of y
+    ,                        !- Minimum Curve Output
+    ,                        !- Maximum Curve Output
+    Temperature,             !- Input Unit Type for X
+    Temperature,             !- Input Unit Type for Y
+    Dimensionless;           !- Output Unit Type
+
+  Curve:Quadratic,
+    EIRCurveFuncPLR,         !- Name
+    1.0,                     !- Coefficient1 Constant
+    0.0,                     !- Coefficient2 x
+    0.0,                     !- Coefficient3 x**2
+    0.0,                     !- Minimum Value of x
+    1.0;                     !- Maximum Value of x
```

Comment:
Sorry for jumping late, i just noticed this PR dropping in develop yesterday and I am interested in the feature, as I have had to model AWHP before including some systems like the one `mentioned` in the PR's original post.

I'm a bit confused by these curves, and they seem really generic to mimic a real world system.

I'm sure the testfiles added are nice to ensure that a simulation runs and for coverage, but it's also an Example File, so does it mimic real world operation?